src/Microsoft.AspNetCore.WebSockets/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.2.0-*",
+  "version": "1.0.0-*",
```

Comment:
Yep. Technically, since they match projects, they don't need to be updated but they should be. I'll do that